Orthtree/include/CGAL/Orthtree.h

```diff
@@ -43,258 +48,222 @@
-    \param point_map property map to access the input points.
-    \param enlarge_ratio ratio to which the bounding box should be enlarged.
     \param traits the traits object.
   */
-  Orthtree(PointRange& point_range,
-           PointMap point_map = PointMap(),
-           const FT enlarge_ratio = 1.2,
-           Traits traits = Traits())
-    : m_traits (traits)
-    , m_range (point_range)
-    , m_point_map (point_map)
-    , m_root(Node(), 0)
-  {
-    Array bbox_min;
-    Array bbox_max;
+  explicit Orthtree(Traits traits) :
+    m_traits(traits),
+    m_node_contents(m_node_properties.add_property<Node_data>("contents")),
+    m_node_depths(m_node_properties.add_property<std::uint8_t>("depths", 0)),
+    m_node_coordinates(m_node_properties.add_property<Global_coordinates>("coordinates")),
+    m_node_parents(m_node_properties.add_property<Maybe_node_index>("parents")),
+    m_node_children(m_node_properties.add_property<Maybe_node_index>("children")) {
 
-    // init bbox with first values found
-    {
-      const Point& p = get (m_point_map, *(point_range.begin()));
-      std::size_t i = 0;
-      for (const FT& x : cartesian_range(p))
-      {
-        bbox_min[i] = x;
-        bbox_max[i] = x;
-        ++ i;
-      }
-    }
+    m_node_properties.emplace();
 
-    for (const Range_type& r : point_range)
-    {
-      const Point& p = get (m_point_map, r);
-      std::size_t i = 0;
-      for (const FT& x : cartesian_range(p))
-      {
-        bbox_min[i] = (std::min)(x, bbox_min[i]);
-        bbox_max[i] = (std::max)(x, bbox_max[i]);
-        ++ i;
-      }
-    }
+    // init bbox with first values found
+    auto bbox = m_traits.construct_root_node_bbox_object()();
 
-    Array bbox_centroid;
-    FT max_length = FT(0);
-    for (std::size_t i = 0 ; i < Dimension::value; ++ i)
-    {
-      bbox_centroid[i] = (bbox_min[i] + bbox_max[i]) / FT(2);
-      max_length = (std::max)(max_length, bbox_max[i] - bbox_min[i]);
-    }
-    max_length *= enlarge_ratio / FT(2);
+    // Determine dimensions of the root bbox
 
-    for (std::size_t i = 0 ; i < Dimension::value; ++ i)
-    {
-      bbox_min[i] = bbox_centroid[i] - max_length;
-      bbox_max[i] = bbox_centroid[i] + max_length;
+    Bbox_dimensions size;
+    for (int i = 0; i < Dimension::value; ++i) {
+      m_bbox_min[i] = (bbox.min)()[i];
+      size[i] = CGAL::Exact_predicates_exact_constructions_kernel::FT((bbox.max)()[i]) - m_bbox_min[i];
     }
 
-    Construct_point_d_from_array construct_point_d_from_array
-      = m_traits.construct_point_d_from_array_object();
-
     // save orthtree attributes
-    m_bbox_min = construct_point_d_from_array(bbox_min);
-    m_side_per_depth.push_back(bbox_max[0] - bbox_min[0]);
-    m_root.points() = {point_range.begin(), point_range.end()};
+    m_side_per_depth.push_back(size);
+    data(root()) = m_traits.construct_root_node_contents_object()();
   }
 
   /// @}
 
-  /// \cond SKIP_IN_MANUAL
```

Comment:
Looks like the corresponding `\endcond` is still present (line 269), either both should be present or both should be absent.
(See also: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.0-Ic-158/logs_master/Orthtree.log)

(Note: around line 160 there looks like to be an empty `\cond` / `\endcond` block).